src/views/OrderPage/OrderPage.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import * as Style from './index';
+import { Header } from '../../shared/components';
+import * as Components from '../../modules/order';
+import Topping from 'modules/order/models/Topping';
```

Comment:
You are mixing default exports and named exports (Header vs Topping). It is preferred not to use `export default` if you can help it.